### DISCUSSION TITLE
v1.1.0-beta.2

### DISCUSSION THREAD

**Discussion by Losses** [Losses/rune#217] [Announcements]
## Features
- Updated translation files by @123Duo3.

## Bug Fixes
- Improved randomness of the cover art wall.
- Enhanced image rendering quality by setting filter quality to high.
- Fixed issue where switching among playback modes was not possible when there were insufficient enabled modes.
- Resolved issue where the navigation back text was not clickable on the bottom side.

**Full Changelog**: https://github.com/Losses/rune/compare/v1.1.0-beta.1...v1.1.0-beta.2

<hr /><em>This discussion was created from the release <a href='https://github.com/Losses/rune/releases/tag/v1.1.0-beta.2'>v1.1.0-beta.2</a>.</em>

**Comment by dheemanthm2003** [comment]
(https://github.com/user-attachments/assets/a873c94a-5b0b-4b57-b5f6-e144240698ae)

I am getting this error after allowing access to music folder and selecting local hard drive (arm64, android 10, realme 2 pro).

**Comment by Losses** [comment]
@dheemanthm2003 As it's shown in the README, android builds are highly experimental. All fs related things are still buggy, we are finding solution to integrate Rust and Android correctly.
